docs/design/README.md

```diff
@@ -0,0 +1,552 @@
+# Twoliter Design Document
+
+## Introduction
+
+Twoliter is a command line tool for building, testing and deploying custom variants of [Bottlerocket](https://github.com/bottlerocket-os/bottlerocket).
+
+### Background
+
+One of Bottlerocket's security features is its immutable root filesystem and lack of package manager.
+Because software cannot be installed on Bottlerocket after the image has been created, the build system supports creating different Bottlerocket *variants*.
```

Comment:
```suggestion
Because software cannot be installed on Bottlerocket after the image has been created, it lacks a traditional package manager.
Instead, the build system supports creating different Bottlerocket *variants*.
```